addon/styles/_variables.scss

```diff
@@ -0,0 +1,49 @@
+//
```

Comment:
Not sure what's up with the spacing in this file, but I'd say go with putting the values just after their names instead of having them lined up some distance away.